.github/workflows/galaxy.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Ansible Galaxy release
+
+on:
+  release:
+    types: [created, edited, published, released]
+  push:
+    tags:
+      - '*'
+
+permissions: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          path: ajnasz.miniflux
+      - name: galaxy
+        uses: robertdebock/galaxy-action@1.2.1
+        with:
+          galaxy_api_key: ${{ secrets.galaxy_api_key }}
```

Comment:
I guess I'll need to add an api key.

Reply:
Yes. Go to https://galaxy.ansible.com/me/preferences and logged in with github to get API key.

To set in Settings > secrets and variables: Repository secrets.
It also seems there is a new option in Settings > Envvironments: galaxy: Environment secrets.